src/registry/registry.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Button, Container, Col, Row } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+export default function Registry() {
+  return (
+    <>
+      {' '}
```

Comment:
If this space is not needed on render, then consider removing. Otherwise, disregard this comment.

Reply:
You can't remove that spacing. It's in a react fragment. I needed that for sizing in an earlier version, not needed anymore. I'll remove the fragment altogether for readability.